Zend/zend_ini.c

```diff
@@ -876,3 +876,23 @@ ZEND_API ZEND_INI_MH(OnUpdateStringUnempty) /* {{{ */
 	return SUCCESS;
 }
 /* }}} */
+
+ZEND_API ZEND_INI_MH(OnUpdateStr) /* {{{ */
+{
+	zend_string **p = (zend_string **) ZEND_INI_GET_ADDR();
+	*p = new_value ? new_value : NULL;
```

Comment:
Can't this be simplified:
```suggestion
	*p = new_value;
```

Reply:
Indeed